core/install_plugin_dependencies.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+find ./cat/plugins -type f -name "requirements.txt" | while read -r req_file; do
+    pip install --no-cache-dir -r "$req_file"
+done
```

Comment:
Hi,
I'm totally new to this repo in which I have no experience and this is just a comment not a review.
Anyway, I thought that this way to handles dependencies could be troublesome. 
Think about having different plugins which use the same library but support a different version, based on the order in which the scripts install the dependencies one or more script cannot be run correctly.
Probably it needs a script, bash or python or something else, which reads all the dependencies and their version and creates a new requirements.txt with the list of libraries and the minimum version or return an error if the versions of the same library are not compatible (jinja<2.0, jinja>=3.X).

These are my 2 cents about this PR.